content/applications/sales/point_of_sale/restaurant/floors_tables.rst

```diff
@@ -2,91 +2,106 @@
 Floors and tables management
 ============================
 
-Add a floor
-===========
+Floor plan
+==========
 
-Once your *Point of Sale* has been configured, select *Table
-Management* under :menuselection:`Point of Sale --> Configuration -->
-Point of Sale`. Then, click on *Floors* to create and name your floor and tables.
+To understand  the layout of your floor plan, keep the following keys in mind:
 
-.. image:: floors_tables/restaurant_03.png
-    :align: center
-    :alt: View of the table management feature. Way to manage and create floors for a pos
+.. image:: floors_tables/plan-understand.png
 
-.. image:: floors_tables/restaurant_04.png
-    :align: center
-    :alt: Backend view of a restaurant floor. Table name and number of sits for each table
+- Table 1: An order has been placed and sent to the kitchen.
+- Table 3: An order of four items has been placed and needs to be sent to the kitchen.
+- Tables 2, 4, and 5: These tables are available.
+- Tables 2, 4, and 5: These tables' total capacity is, respectively, 2, 4, and 8 people.
+- Table 1: The table of two is full.
+- Table 3: The table of four is taken by one person.
 
-.. note::
-   Don’t forget to link your floor to your point of sale.
-
-Add tables
-==========
+Configuration
+=============
 
-To add tables, you can also open your PoS interface to see your
-floor(s).
+From the POS settings
+---------------------
 
-.. image:: floors_tables/restaurant_05.png
-    :align: center
-    :alt: View of the floors menu to manage several floors at the same time
+To create floors and tables from the backend, :ref:`access your POS settings
+<configuration/settings>` and click :guilabel:`→ Floors` under the :guilabel:`Restaurant Mode`
+section's :guilabel:`Floors & Tables Map` category.
 
-Then, click on *Edit Mode* (pencil icon on the upper right corner) to be allowed to
-create, move, modify tables, etc.
+Then, click :guilabel:`New` to create a floor, name it, select the related point(s) of sales,
+and click :guilabel:`Add a line` to create some tables. Name the table and assign a number of seats.
+You can also link it to an appointment resource to make the table bookable. Once done, click
```

Comment:
It would be nice to put a link to show how to do so.

Reply:
It would be great, but there's currently no documentation about the Appointment app, I asked Tiku and he said he was fine with me adding a  doc page but I haven't done it yet.

However, I'll put on the task to come back here and add a link when the page is created